modules/ROOT/pages/configuration/configuration-settings.adoc

```diff
@@ -2239,8 +2239,8 @@ label:enterprise-edition[Enterprise Edition] label:dynamic[Dynamic]
 [frame="topbot", stripes=odd, grid="cols", cols="<1s,<4"]
 |===
 |Description
-a|Enables or disables tracking of how much time a query spends actively executing on the CPU. Calling `SHOW TRANSACTIONS` will display the time, but not in the _query.log_. +
-If you want the CPU time to be logged in the _query.log_, set `db.track_query_cpu_time=true` and `db.logs.query.time_logging_enabled=true` .
+a|Enables or disables tracking of how much time a query spends actively executing on the CPU. Calling `dbms.listQueries` will display the time. +
+This can also be logged in the query log by using `dbms.logs.query.time_logging_enabled`. .
```

Comment:
out of curiosity, is it clear enough that you have to set  `dbms.logs.query.time_logging_enabled=true`?

Reply:
This is from the code base.